apps/makescene/makescene.cc

```diff
@@ -834,6 +932,26 @@ import_images (AppSettings const& conf)
         view->set_id(id);
         view->set_name(remove_file_extension(fname));
 
+        std::string acfname;
+        if (find_corresponding_cam_file(dir, i, &acfname)) {
```

Comment:
I don't understand your implementation of this function. Why not just

    string cam_filename = find_corresponding_cam_file(afname);
    if (!cam_filename.empty()) ...